content/influxdb/v2.0/upgrade/v1-to-v2.md

```diff
@@ -12,33 +12,181 @@ aliases:
+Consider whether you need to address any of the following before upgrading.
+
+<!--
+POSSIBLE ALT:
+Before upgrading to InfluxDB 2.0, review this section to see if you use these features:
 
-{{% warn %}}
-Back up all data before upgrading with `influx upgrade`.
-{{% /warn %}}
+- Continuous queries
+- ...
+
+If you use one or more of these features, complete the instruction below to ensure a successful upgrade. We plan to address these items in the near future; as such, you may prefer to hold off on upgrading.
+-->
+
+### Available operating system, container, and platform support
+
+InfluxDB 2.0 is currently available for MacOS and Linux.
+Docker images, ARM builds, and Windows builds are not currently available.
+These are planned for subsequent releases.
 
 {{% note %}}
-The 1.x `_internal` database is not migrated with the `influxd upgrade` command.
+We are working on the upgrade process for users running Docker and 1.8.x.
+Hold off on upgrading for now.
 {{% /note %}}
 
-1. [Download InfluxDB OSS 2.0](https://portal.influxdata.com/downloads/),
-   unpackage the InfluxDB binaries, and then place them in your `$PATH`.   
+There is no plan for a 32-bit build of InfluxDB 2.0.
+
+### Continuous queries
+
+If you use continuous queries, review them prior to upgrading by running the `show continuous queries` command:
+Run the following:
+
+{{< keep-url >}}
+```sh
+$ influx
+Connected to http://localhost:8086 version 1.8.3
+InfluxDB shell version: 1.8.3
+> show continuous queries
+```
+
+Save your continuous queries output to use later.
+
+Continuous queries are replaced by tasks in InfluxDB 2.0
+To convert your continuous queries to Flux, see [Get started with InfluxDB tasks](/influxdb/v2.0/process-data/get-started/).
+
+### Supported protocols
+
+If you're using one of the [supported protocols](/influxdb/v1.8/supported_protocols/) in InfluxDB 1.x (CollectD, Graphite, OpenTSDB, Prometheus, UDP),
+InfluxDB 2.0 doesn't support these protocols directly.
+Use [Telegraf](/telegraf/v1.16/) to translate from the source protocol to InfluxDB 2.0.
+
+### Kapacitor
+
+You can continue to use Kapacitor with InfluxDB OSS 2.0 under the following scenarios:
+
+- Kapacitor Batch-style TICKscripts work with the 1.x read compatible API
+  Existing Kapacitor user credentials should continue to work using the [1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+- Kapacitor Stream-style TICKscripts will not work using the [subscription API](/influxdb/v1.8/administration/subscription-management/).
+  (There is no subscriptions API in InfluxDB 2.0.)
+  We recommend writing data directly to *both* Kapacitor and InfluxDB to allow stream tasks to continue to work.
+  To do this, configure two [InfluxDB output plugins](/telegraf/v1.16/plugins/#influxdb): one for Kapacitor and one for InfluxDB 2.0.
+  If you use other data sources to send data to InfluxDB, Telegraf can be used as an intermediate layer to write to both Kapacitor and InfluxDB.
```

Comment:
InfluxDB 2.0 has no subsriptions API and does not support Kapacitor stream tasks. To continue using stream tasks, write data directly to both InfluxDB and Kapacitor. Use **Telegraf** and its [InfluxDB output plugin](/{{< latest "telegraf" >}}/plugins/#influxdb) to write to Kapacitor and the [InfluxDB v2 output plugin](/{{< latest "telegraf" >}}/plugins/#influxdb_v2) to write to InfluxDB v2.

##### Example Telegraf configuration
```toml
# Write to Kapacitor
[[outputs.influxdb]]
  urls = ["http://localhost:9092"]
  database = "example-db"
  retention_policy = "example-rp"

# Write to InfluxDB 2.0
[[outputs.influxdb_v2]]
  urls = ["http://localhost:8086"]
  token = "$INFLUX_TOKEN"
  organization = "example-org"
  bucket = "example-db/example-rp"
```

Reply:
they may also decide to use the 1.0 output plugin for InfluxDB given the starting point?

Reply:
That's true, they definitely could. In fact, that's probably the better way to go, since it will create the DBRP mappings.